README.md

```diff
@@ -38,6 +41,15 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list. Feel fr
 <br>
 > <h3> Libraries</h3>
 
+#### Loaders
+
+- [Babel Loader](https://github.com/babel/babel-loader) - Webpack plugin for Babel.
+ - Maintainer: `Babel Team`  - [![Github](https://github.com/encharm/Font-Awesome-SVG-PNG/blob/master/black/svg/github-square.svg)](https://github.com/babel)
```

Comment:
Hmmm, this doesn't appear to be loading :-(

<img width="365" alt="screen shot 2016-08-01 at 4 06 59 pm" src="https://cloud.githubusercontent.com/assets/1500684/17310493/04f81524-5802-11e6-8307-8ecc67b5a14c.png">